test/api.test.js

```diff
@@ -0,0 +1,15 @@
+const { getMovieMock, postMovieMock } = require('../mock/apiMock.js');
+
+describe('Testing the API Request and response', () => {
+  test('Retrieving data from the API', async () => {
+    const data = await getMovieMock();
+    expect(data.title).toEqual('delectus aut autem');
+  });
+
+  test('Posting data to the API', async () => {
+    const post = await postMovieMock(1, 'Hello World', false);
+    expect(post.title).toEqual('Hello World');
+    expect(post.userId).toEqual(1);
+    expect(post.completed).toBeFalsy();
+  });
+});
```

Comment:
- [ ] some of your tests are failing kindly fix this
![image](https://user-images.githubusercontent.com/66097371/227654025-85c76036-4d58-449f-add2-9a731ca9c9a3.png)